packages/css/src/logo/README.md

```diff
@@ -0,0 +1,35 @@
+# Logo
+
+Het logo van de gemeente bestaat uit een beeldmerk (3 Andreaskruisen) en een woordmerk (Gemeente Amsterdam).
+Het logo heeft een vaste kleur, grootte en positie en staat op al onze websites.
+
+## Specificaties
+
+Gebruik altijd het online logo.
+Het Nederlandse logo voor Nederlandstalige websites en anderstalige websites behalve Engels.
+Het Engelstalige logo gebruiken we voor Engelstalige websites.
```

Comment:
Do we have the English logo? If so, we should add it to the list. If not, maybe we should mention that here?

Reply:
[We do](https://amsterdam.nl/stijlweb/basiselementen/logo-gemeente-amsterdam/#hba498e5c-2388-464f-b769-7622a5d32bd1), but as far as I can see it’s only available in a non-web format. I suggest we create a separate task for this.

Reply:
We can add that link for now?

Reply:
Something like this?

```suggestion
Het Engelstalige logo gebruiken we voor Engelstalige websites. Dit logo moet nog worden toegevoegd, [je kunt hier een plaatje van het logo vinden](https://amsterdam.nl/stijlweb/basiselementen/logo-gemeente-amsterdam/#hba498e5c-2388-464f-b769-7622a5d32bd1). 
```

Reply:
I suggest slightly different wording, and a sentence per line in Markdown.

```suggestion
Het Engelstalige logo gebruiken we voor Engelstalige websites.
Deze afbeelding wordt later toegevoegd aan het design system.
In de tussentijd kun je het downloaden vanaf [Stijlweb](https://amsterdam.nl/stijlweb/basiselementen/logo-gemeente-amsterdam/#hba498e5c-2388-464f-b769-7622a5d32bd1).
```

Reply:
Small tweak:

```suggestion
Het Engelstalige logo gebruiken we voor Engelstalige websites.
Dit logo wordt later toegevoegd aan het design system.
In de tussentijd kun je een afbeelding van dit logo downloaden vanaf [Stijlweb](https://amsterdam.nl/stijlweb/basiselementen/logo-gemeente-amsterdam/#hba498e5c-2388-464f-b769-7622a5d32bd1).
```